docs/VnVPlan/VnVPlan.tex

```diff
@@ -347,6 +347,47 @@ \subsubsection{Area of Testing2}
 
 ...
 
+\subsubsection{Operational and Environmental Requirements}
+
+\begin{enumerate}
+\item{NFRT-OER1\\}
+
+Type: Functional, Dynamic, Manual
+
+Initial State: Application is not yet installed on the device.
+                
+Input: The user will install and run the application on various devices.
+                
+Output: The application will be successfully installed. The application is operational.
+                
+How test will be performed: The team will install the application on various mobile devices running Android on version 13 (IASR2) and iOS on version 16 (IASR1). The team will also test on various tablet devices running Android or iPadOS. The team will confirm that all functionalities work as expected on each device by conducting variety of actions and system events.
+
+\item{NFRT-OER2\\}
+
+Type: Functional, Dynamic, Manual
+
+Initial State: The application is installed on device that have operable camera peripherals and access to the internet.
+                
+Input: The user will attempt to use application features that require a camera and/or internet access.
+                
+Output: The application will successfully operate the induced features that require the use of camera and/or internet.
+                
+How test will be performed: The team will test the application on devices with and without camera peripherals to validate (WER1). The team will disconnect devices from the internet to test offline functionality. The team will then reconnect the device to the internet to test online features (WER2). The team will confirm that all functionalities work on devices that have access to camera peripherals and internet access.
+
+\item{NFRT-OER3\\}
+
+Type: Static, Dynamic, Automated
+
+Initial State: The application is ready for new releases or any updates.
+                
+Input: The user will attempt to execute test suites against new releases and updates.
```

Comment:
Maybe instead of the user, just say "test suites will be run against new releases and updates". Saying "the user" here doesn't make sense in my opinion since the user would not be executing tests in this capacity.

Reply:
resolved